.github/workflows/color-check.yml

```
@@ -26,6 +27,18 @@ jobs:
           git fetch origin $TARGET_BRANCH
 
       - name: 🎨 Check colors
+        id: color-check
         run: |
           svgFiles=$(git diff origin/$TARGET_BRANCH --diff-filter=ACMRTUX  --name-only | grep '.svg$')
           npx svg-color-linter --config material-colors.yml ${svgFiles}
+
+      - name: 🏷️ Manage label based on color check result
+        continue-on-error: true
```

This will never be executed, because the step before will stop the execution of this workflow. The `continue-on-error` is related to the current step and not to the workflow in general ([see the docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error)).

Instead I'd recommend to use [`if: {{ always() }}`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always) to execute some actions after a previous step has failed.

```suggestion
        if: ${{ always() }}
```